app/src/test/java/com/myfitnesstracker/NutritionDataUnitTest.kt

```diff
@@ -22,18 +22,21 @@ class NutritionDataUnitTest {
 
 
     private fun givenAFeedOfNutritionDataAreAvailable() {
-        mvn = MainViewModel()
+        //Changed from mvn to mvm
+        mvm = MainViewModel()
     }
 
 
     private fun whenSearchForPho() {
-        mvn.fetchNutritionInfo("pho")
+        //Changed from mvn to mvm
+        mvm.fetchNutritionInfo("pho")
     }
 
 
     private fun thenResultContainsPhoNutritionInfo() {
         var phoFound = false;
-        mvn.nutrition.observeForever {
+        //Changed from mvn to mvm
+        mvm.nutrition.observeForever {
```

Comment:
Changed mvn to mvm so the unit test knew you were referencing the MainViewModel
-I did not change it on the other tests